LiskHQ/lisk-hub
# Migrate account switcher to a page - Closes #530, #338

### What was the problem?
Accounts view is currently displayed in a modal dialog, and should reside on a new page accessible by /accounts url

### How did I fix it?
- provide a new separate route, 
- remove dialog code related to savedAccounts
- adapt some styles, regarding z-indexes
- generate unique ids for gradients and shapes referenced in accountVisual
- adapt fontsizes to px, which were wrong compared to reference lisk.io when we changed rem to px

### How to test it?
go to main/dashboard, click on accountVisual menu, 
- try to add new account, remove it, re-add it, go back in history, reload page, 
- everything should work as expected, but now url should be /accounts.

### Review checklist
- The PR solves #530, #338 
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices
